fb_reels_publishing_api_sample/index.js

```diff
@@ -225,6 +225,78 @@ app.post("/uploadReels", function (req, res) {
     });
 });
 
+/**
+ * List Uploaded Reels of the Selected Page
+ * If No page is selected, stay on the Upload Page and display the error
+ * Else, display all relevant reels of that were previously uploaded for the selected page
+ */
+app.post("/listUploadedVideos", async function(req, res) {
```

Comment:
only one nit: this should be a get (we are actually not posting anything, rather getting the list)